lib/beaker-hostgenerator/hypervisor.rb

```diff
@@ -45,7 +45,8 @@ def self.builtin_hypervisors()
         'vagrant' => BeakerHostGenerator::Hypervisor::Vagrant,
         'vagrant_libvirt' => BeakerHostGenerator::Hypervisor::Vagrant,
         'docker' => BeakerHostGenerator::Hypervisor::Docker,
-        'abs' => BeakerHostGenerator::Hypervisor::ABS
+        'abs' => BeakerHostGenerator::Hypervisor::ABS,
+        'hcloud' => BeakerHostGenerator::Hypervisor::Hcloud
```

Comment:
Does Rubocop allow a trailing comma?
```suggestion
        'hcloud' => BeakerHostGenerator::Hypervisor::Hcloud,
```